src/util.cpp

```diff
@@ -45,13 +45,13 @@ hipblasCreate(hipblasHandle_t* handle)
 {
     if(handle != nullptr)
     {
-        // HIP supports mutile backends hence query current backend name
-        auto backendName = hipGetBackendName();
         // Obtain the handles to the back handlers.
-        unsigned long handles[4];
-        int           nHandles = 4;
-        hipGetBackendNativeHandles((uintptr_t)NULL, handles, &nHandles);
-        *handle = H4I::MKLShim::Create(handles, nHandles, backendName);
+        int nHandles;
```

Comment:
These are backward compatibility breaking changes. We should add a check to verify chipStar is compatible. Think of a situation where chipStar runtime is old but hipBlas is new.